tasks/level-1/1.6.1.1.yml

```diff
@@ -3,22 +3,21 @@
 
 # 1.6.1.1 Ensure SELinux is not disabled in bootloader configuration (Scored)
 
-- name: Install the SE Linux requirements - libselinux-python
+- name: 1.6 - Install the SE Linux requirements - libselinux-python
   yum:
     name: libselinux-python
     state: installed
 
-- name: Install the SE Linux requirements - policycoreutils-python
+- name: 1.6 - Install the SE Linux requirements - policycoreutils-python
   yum:
     name: policycoreutils-python
     state: installed
 
 - name: 1.6.1.1 Ensure SELinux is not disabled in bootloader configuration (Scored)
-  selinux:
-    policy: targeted
-    state: enforcing
+  command: egrep -c 'selinux=0|enforcing=0'
+  register: line_matched
+  failed_when: line_matched.stdout != '0'
```

Comment:
So far as I can tell this change won't ensure that SELinux is not disabled in bootloader config. It will simply fail if it is not already enforced.